src/main/resources/view/PersonListCard.fxml

```diff
@@ -26,6 +26,7 @@
           </minWidth>
         </Label>
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
+        <FlowPane fx:id="lead" />
```

Comment:
The positioning for the `lead` tag looks a bit off. Maybe can set the parent `HBox` alignment to `BOTTOM_LEFT` to align the elements to the bottom.